ghost/core/core/server/services/url/Resources.js

```diff
@@ -297,6 +299,25 @@ class Resources {
 
         const resourceConfig = _.find(this.resourcesConfig, {type: type});
 
+        if (resourceConfig && Object.keys(model._changed).length) {
```

Comment:
This code is the 🥩  of the issue. It bypasses the URL update process whenever only non-essential properties were changed.

The 'updated_at' is taken into account separately, so we can emit an event about the resource update, which sitemaps service needs.